enterprise/server/remote_execution/platform/platform.go

```diff
@@ -210,6 +210,15 @@ type ExecutorProperties struct {
 	DefaultXcodeVersion     string
 }
 
+func (p *ExecutorProperties) SupportsIsolation(c ContainerType) bool {
+	for _, s := range p.SupportedIsolationTypes {
+		if s == c {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
nit: This feels like a future refactor `p.SupportedIsolationTypes` into `map[]struct{}`, instead of a slice, for easier lookup.

I think there is a `func contains()` in this same file that does something similar.

Reply:
removed contains() in favor of the new `SupportsIsolation()`